src/platform_impl/macos/view.rs

```diff
@@ -514,6 +524,7 @@ declare_class!(
 
             let ime_related = was_in_preedit || now_in_preedit || text_commited;
 
+            // todo - wouldn't self.state.forward_key_to_app always be false?
```

Comment:
Nope, `interpretKeyEvents:` calls `doCommandBySelector:` internally, so that may change it